### ISSUE TITLE
Execution killed with Multi Layer Perceptrion

### ISSUE THREAD

**Issue by DoktorMoreau** [fchollet/keras#3492]
Hello,

I try to implement a classification with a Multi Layer Perceptron (MLP). 
My input consist of 
1) picture (1dimensional vector with 450x450 px),
2) label age (1dimensional vector), over 50 or not.

The compilation works, but when I try to start the evaluate the model it crashes with a 'killed'.  So what can I do? Maybe the picture is too big, should I reshape it?
Thanks a lot. My code is below.
# kr_stl_age_mlp.py

from **future** import print_function
import numpy as np
import data_set

np.random.seed(1337)

from keras.datasets import mnist
from keras.models import Sequential
from keras.layers.core import Dense, Dropout, Activation
from keras.optimizers import SGD, Adam, RMSprop
from keras.utils import np_utils

batch_size = 128
nb_classes = 2
nb_epoch = 2

trainImage, trainAge, trainGender, testImage, testAge, testGender = data_set.read_data_set()

print(trainImage.shape[0], 'trainImage')
print(testImage.shape[0], 'testImage')

model = Sequential()
model.add(Dense(512, input_shape=(228962,)))
model.add(Activation('relu'))
model.add(Dropout(0.2))
model.add(Dense(512))
model.add(Activation('relu'))
model.add(Dropout(0.2))
model.add(Dense(2))
model.add(Activation('softmax'))

model.summary()

model.compile(loss='categorical_crossentropy',
              optimizer=RMSprop(),
              metrics=['accuracy'])

history = model.fit(trainImage, trainAge,
                    batch_size=batch_size, nb_epoch=nb_epoch,
                    verbose=1, validation_data=(testImage, testAge))

score = model.evaluate(testImage, testAge, verbose=0)
print('score:', score)

**Comment by hqsiswiliam** [comment]
You can try to reduce the batch size for training process.
